src/fondant/compiler.py

```diff
@@ -404,7 +407,6 @@ def compile(
         @self.kfp.dsl.pipeline(name=pipeline.name, description=pipeline.description)
         def kfp_pipeline():
```

Comment:
This looks identical to the one above. Should we lift it to a base class?

Reply:
I have a feeling that the compiler for vertex and kfpV2 can be the same. I would like to test both more and then we can merge (or abstract)